### DISCUSSION TITLE
No color and image in PDF

### DISCUSSION THREAD

**Discussion by lateliercom** [frappe/print_designer#142] [Q&A]
Hello,
I have installed the print designer app, all working good in the admin view :
![Capture d’écran 2024-01-25 à 12 34 53](https://github.com/frappe/print_designer/assets/157702430/6ab68ae3-df27-49fd-a804-c3df9419adb6)

But  when I go to quotation, i have this (no images, not colorised text):
![Capture d’écran 2024-01-25 à 12 35 36](https://github.com/frappe/print_designer/assets/157702430/b5ac40ef-b383-49a1-acc0-58380f6b2bab)

I tried to reinstall but the issus is still here... If you have any idea how to do ?

PS : With the basic design i also not having images in pdf

**Comment by maharshivpatel** [comment]
@lateliercom you need to install correct wkhtmltopdf version

 - Frappe / Print Designer requires at least  [version 0.12.5 with patched qt](https://frappeframework.com/docs/v14/user/en/installation#pre-requisites)

    - Install correct version by following [installation guide](https://frappeframework.com/docs/v14/user/en/installation) and confirm : 

    ```
    wkhtmltopdf --version
    ```
    - output should be similar to `wkhtmltopdf 0.12.6 (with patched qt)`

**Reply by lateliercom** [reply]
Thanks for the reply, i have this : `wkhtmltopdf 0.12.6.1 (with patched qt)`

Before SSL added, i had the color text, but no images. And after SSL both are gone...

**Reply by maharshivpatel** [reply]
@lateliercom did you upload images from print designer ? and did you make them private ?

there was one issue with color. can you please update print designer to the latest version

**Reply by lateliercom** [reply]
For images : I use the one I allready upload (private and plublic didin't work), if I want to upload I have this error :

![Capture d’écran 2024-01-30 à 12 32 31](https://github.com/frappe/print_designer/assets/157702430/26a3c842-2cb7-476a-a256-bc4857836c57)

For color : Its work now with the last version 👍

**Reply by lateliercom** [reply]
@maharshivpatel Hello, any update on the images bug ?

**Reply by lateliercom** [reply]
@maharshivpatel I can upload images now, I fix the bug.

But i can't see them in private or public, and png or jpg are same. Do you know how can I fix it  ? :)
